Tests/SwiftParserTest/translated/StringLiteralEofTests.swift

```diff
@@ -65,7 +69,10 @@ final class StringLiteralEofTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "invalid escape sequence in literal"),
         DiagnosticSpec(locationMarker: "2️⃣", message: #"expected '"' to end string literal"#, fixIts: [#"insert '"'"#]),
-      ]
+      ],
+      fixedSource: ##"""
+        _ = "foo \"
```

Comment:
We should be doing something about the `\` because this string literal is still not valid since the closing `"` is escaped.